File: csharp/DVSClient/DVSClient/exceptions/RestApiInterruptionOrExecutionException.cs
namespace DVSClient.Exceptions
{
    /// <summary>
    /// Represents an exception that is thrown when a REST API call is interrupted or fails during execution.
    /// </summary>
    public class RestApiInterruptionOrExecutionException : EDVSException
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="RestApiInterruptionOrExecutionException"/> class with a reference to the exception that caused the error.
        /// </summary>
        /// <param name="e">The exception that caused the REST API interruption or execution failure.</param>
        public RestApiInterruptionOrExecutionException(Exception e) : base(e)
        {
        }
    }
}

File: java/src/main/java/com/experian/dvs/client/exceptions/RestApiInterruptionOrExecutionException.java
package com.experian.dvs.client.exceptions;

/**
 * Represents an exception that is thrown when a REST API call is interrupted or fails during execution.
 */
public class RestApiInterruptionOrExecutionException extends EDVSException {

    /**
     * Initializes a new instance of the {@link RestApiInterruptionOrExecutionException} class with a reference to the exception that caused the error.
     *
     * @param e The exception that caused the REST API interruption or execution failure.
     */
    public RestApiInterruptionOrExecutionException(Exception e) {
        super(e);
    }
}

